app/assets/stylesheets/reskin/moneyhelper_styling_common.scss

```diff
@@ -0,0 +1,210 @@
+// -- Headers ----------------------------------------------------------------------
+h1,h2,h3,h4,h5,h6{
+    font-family: 'Roobert-Bold', 'Times';
+    font-style: normal;
+    font-weight: bold;
```

Comment:
The declarations for `font-style` and `font-weight` are strictly speaking redundant given we are declaring the weight in the `font-family` declaration. Though these would serve as a style for a fallback font here it would be more efficient to declare them in the `@font-face` definition.